docs/pages/cloud/faq.mdx

```diff
@@ -67,14 +69,14 @@ cluster on AWS.
 ### Will Teleport be automatically upgraded with each release?
 
 We will be upgrading the preview version of Teleport Cloud automatically.
-We will add an option to trigger upgrades manually and automatically
-within a configurable maintenance window.
+
+We will add an option to trigger upgrades manually and automatically within a configurable maintenance window.
 
 ### Does your SOCII report include Teleport Cloud?
 
-Our current SOCII Type2 report covers our organization, Teleport and Gravity OSS and Enterprise versions.
-We are pursuing SOC2 for cloud and expect to receive our report at the end of Q2 2021
-after our 6 month audit period is completed.
+Our current SOCII Type2 report covers our organization, Teleport and Gravity Open Source, and Enterprise Enterprise.
+
+We are pursuing SOC2 for Cloud and expect to receive our report at the end of Q2 2021 after our 6 month audit period is completed.
```

Comment:
Update with latest SOC2 info here.

Reply:
Yeah, let's - at least for 7.0: https://github.com/gravitational/teleport/issues/7186 - should we update this for 6.2 as well?

Reply:
Hmmm ... We can also remove this section and link to a new Architecture/Security article section. (Or refactor it into an include so it lives in both places.) 

You have a preference about the best way here? Will we have video or blog content to support that? Seems like a cool thing to detail a bit - plus there seems to be some confusion about Enterprise and Open Source offerings presently. Maybe something that tackles all of those?